modules/bm_sim/include/bm_sim/lpm_trie.h

```diff
@@ -1,91 +0,0 @@
-/* Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
Please leave this file as it is (it is just a C++ wrapper around some C code I wrote a while ago) and use LPMTrie in your LPM lookup structure implementation.